src/fromMovingPoints.js

```diff
@@ -1,6 +1,8 @@
 import { translate } from './translate'
 import { applyToPoint } from './applyToPoint'
-import { scale, rotate, compose } from './index'
+export { rotate } from './rotate'
+export { scale } from './scale'
+export { compose } from './transform'
```

Comment:
I understand this issue, but I think that the right word here is `import` rather then `export`

Reply:
Ah, right 🙃 Fixed!